bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -251,9 +251,13 @@ public HTableDescriptor getTableDescriptor(TableName tableName)
 
     try {
       return tableAdapter.adapt(bigtableTableAdminClient.getTable(request));
+    } catch (StatusRuntimeException e) {
+      if (e.getStatus().getCode() == Status.NOT_FOUND.getCode()) {
```

Comment:
The are not actually ==.  A status can have a whole bunch of stuff in it that will make it not ==...

see https://github.com/grpc/grpc-java/blob/572f4332a75367e0d0e1cb096198839b68787670/core/src/main/java/io/grpc/Status.java#L353